tests/test_install.py

```diff
@@ -61,7 +61,19 @@ def test_install_filename_glob(self):
         """Test that the pkg to be installed can be specified by fname glob."""
         self.base.install("*/there")
         (installed, _) = self.installed_removed(self.base)
-        self.assertCountEqual(map(str, installed), ('trampoline-2.1-1.noarch',))
+        self.assertCountEqual(map(str, installed),
+                              ('trampoline-2.1-1.noarch',))
+
+        self.base.install("/all/*/there")
+        (installed, _) = self.installed_removed(self.base)
+        self.assertCountEqual(map(str, installed),
+                              ('trampoline-2.1-1.noarch',))
+
+    def test_install_provide_glob(self):
```

Comment:
Could you please move the method so that the methods are sorted alphabetically (that's our convention)? I also that that it should be rather ```test_install_provide_slash``` (there is no glob in the provide) but nevermind....